### PR TITLE
Upgrade lerna to latest release

### PR DESCRIPTION
**Pull request by justus-camp**
## Description

This PR upgrades Lerna to its latest release now that it's being actively maintained. It leaves the new "useNx" flag false and should have no effect other than potentially removing some security vulnerabilities present in Lerna's dependencies. 

## Reviewer Guidance
Not 100% sure if I'm supposed to include the package-lock and lerna-package-lock changes, but CI wouldn't build without it so I assume it's necessary?

## Does this introduce a breaking change?
No

### CODE REVIEW

File: lerna-package-lock.json [hunk 1]
```diff
@@ -28809,7 +32518,12 @@
 		"is-lambda": {
 			"version": "1.0.1",
 			"resolved": "https://registry.npmjs.org/is-lambda/-/is-lambda-1.0.1.tgz",
+<<<<<<< HEAD
```
Comment (CraigMacomber) [inline lerna-package-lock.json:32521]: Looks like you messed up resolving the conflicts. I recommend recreating this change from scratch (rever the lock file to how it is on main, delete your node modules folder, merge in main, npm i, make sure the lock file is unchanged, then update lerna again and install.

Comment (justus-camp) [reply lerna-package-lock.json:32521]: Ah I've seen VSCode do this to me before. Let me fix it up.

Comment (justus-camp) [reply lerna-package-lock.json]: I'll re-do from scratch to make sure, but I just forgot to hit save after resolving conflicts

Comment (CraigMacomber) [reply lerna-package-lock.json]: Ya, it's gotten me with that before too. VS code has clarity issues around which things save and which don't.

**Review by curtisman — APPROVED** [review]

**Pull request merged**